app/src/main/java/net/gsantner/markor/ui/SearchOrCustomTextDialogCreator.java

```diff
@@ -282,7 +282,6 @@ public static void showSttKeySearchDialog(final Activity activity, final CharSeq
             dopt.titleText = isProjects ? R.string.project : R.string.context;
         }
         dopt.searchHintText = R.string.search_or_custom;
-        dopt.iconsForData = Collections.nCopies(dopt.data.size(), R.drawable.ic_search_black_24dp);
```

Comment:
@gsantner I thought you liked this ?!

It makes it very clear that we are in the search dialog and not in the insert dialog

Reply:
I liked the line of code, but in use it looked bad. 10 identical search icons are bloat

Reply:
Ah. Ok.

Other options are:

1. Dialog level icon (relatively simple addition, this is my preference)
2. Change the dialog title text

Reply:
Why do you see the need for a change? It's not that much bad IMO

Reply:
I would change it to 'Search tag' or 'search context'

Insert context is activated by short press, search context is activated by long press. 

Right now those two dialogs look identical.